README.md

```diff
@@ -1,94 +1,51 @@
-
-## Code scaffolding
 
-Run `nx g @nrwl/react:component my-component --project=my-app` to generate a new component.
+<p align="center"><img src="https://beta.govrn.app/assets/govrn-logo.344f0de2.png" width="350"></p>
 
-## Build
-
-Run `nx build my-app` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
-
-## Running unit tests
-
-Run `nx test my-app` to execute the unit tests via [Jest](https://jestjs.io).
-
-Run `nx affected:test` to execute the unit tests affected by a change.
-
-## Running end-to-end tests
-
-Run `nx e2e my-app` to execute the end-to-end tests via [Cypress](https://www.cypress.io).
-
-Run `nx affected:e2e` to execute the end-to-end tests affected by a change.
-
-## Understand your workspace
-
-Run `nx graph` to see a diagram of the dependencies of your projects.
-
-## Further help
-
-Visit the [Nx Documentation](https://nx.dev) to learn more.
-
-
-
-## ☁ Nx Cloud
-
-### Distributed Computation Caching & Distributed Task Execution
-
-<p style="text-align: center;"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-cloud-card.png"></p>
-
-Nx Cloud pairs with Nx in order to enable you to build and test code more rapidly, by up to 10 times. Even teams that are new to Nx can connect to Nx Cloud and start saving time instantly.
-
-Teams using Nx gain the advantage of building full-stack applications with their preferred framework alongside Nx’s advanced code generation and project dependency graph, plus a unified experience for both frontend and backend developers.
-
-Visit [Nx Cloud](https://nx.app/) to learn more.
+# GovrnMonorepo
+ Our monorepo uses [Nx](https://nx.dev/getting-started/intro) as a build system and for scaffolding.
+
+### Postgres
+Install `postgresql@13` on your OS. It's required by [Prisma](./apps/protocol-api/) to operate.
+Here's a [gist](https://gist.github.com/amrro/e996f84610f074bc2d734f52356be01f) for macOS.
+
+- Install `postgresql@13` binaries.
+- Define a new PostgreSQL user account using `createuser`:
+
+  ```createuser --interactive --pwprompt```
+- To use `postgresql` into the project, you need the `DATABASE_URL` to the **Database Cluster** on your OS. 
+  `DATABASE_URL=postgresql://<username>:<password>@localhost:5432/govrn`
+- Finally, run `yarn nx run protocol-api:migrate`. This will create PostgreSQL database cluster called `govrn`
+  Your database is now in sync with your [schema](./apps/protocol-api) @ `apps/protocol-api/prisma/schema.prisma`.
+- You can acess the `govrn` database cluster using `psql govrn` or using [pgcli](https://github.com/dbcli/pgcli) for auto-completion.
+
+## Quick Start.
+- `yarn install`
+- `yarn nx run protocol-api:serve` to start GraphQL server.
+- If you want to run for example [Linear Sync Job](./apps/linear-sync-job/README.md): `yarn nx run linear-sync-job:serve` 
```

Comment:
Instead of these specific app comands maybe we link to each projects readme and have another quickstart section there.